From 6f88f0a6802f2591b804d7c604c3ac0b589d3515 Mon Sep 17 00:00:00 2001
From: yvanliang <yvan6liang@outlook.com>
Date: Fri, 9 Oct 2026 15:42:29 +0800
Subject: [PATCH 51/60] GSI Android 17 compatibility:
 0053-Not-all-kernel-support-splicing-pipes-or-is-it-socke.patch

Change-Id: Idbad4e975f5ebc146f4f3c239c5cb671fde56728
---
 core/java/android/os/FileUtils.java | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/java/android/os/FileUtils.java b/core/java/android/os/FileUtils.java
index e02e14764..1c0a21a76 100644
--- a/core/java/android/os/FileUtils.java
+++ b/core/java/android/os/FileUtils.java
@@ -482,7 +482,6 @@ public final class FileUtils {
                     return copyInternalSpliceSocket(in, out, count, signal, executor, listener);
                 }
             } catch (ErrnoException e) {
-                throw e.rethrowAsIOException();
             }
         }
 
